pkg/vulnsrc/vulnsrc.go

```diff
@@ -218,18 +208,20 @@ func (o lightOptimizer) Optimize() error {
 
 func (o lightOptimizer) lightOptimize(cveID string, tx *bolt.Tx) error {
 	// get correct severity
-	severity, vendorSeverity, _, _, _, _, _ := getDetailFunc(cveID)
-	vuln := types.Vulnerability{
-		VendorSeverity: vendorSeverity,
+	vuln := getDetailFunc(cveID)
+	lightVuln := types.Vulnerability{
+		VendorSeverity: vuln.VendorSeverity,
 	}
 
+	severity, _ := types.NewSeverity(vuln.Severity)
```

Comment:
Is it fine to use deprecated `vuln.Severity` here and ignore the returned error?

Reply:
As you pointed out, it is not clear. I added comments so that it would be easier-to-understand.
d0c6888

The severity is converted in GetDetail once and this line just reconverts it. It means it never happens an error. Also, we have to keep this variable for backward compatibility.
https://github.com/aquasecurity/trivy-db/pull/54/files/4f9f96510a024b9b9249a7d0f812faa01004b7d4#diff-e04e5beecc49b71f5fefdc5ce12c843fR36